.github/workflows/build_and_test_debug_client.yml

```diff
@@ -44,11 +43,27 @@ jobs:
           files: ./client/src/output/coverage/www/coverage-final.json
           flags: unittests, www
 
+  backend_test:
+    name: Go Backend Test
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: '${{ github.workspace }}/go.mod'
+
+      - name: Test Tun2socks Backend
+        run: go test -v -race -bench=. -benchtime=100ms ./client/...
```

Comment:
Remove the `-v`. It actually makes it harder to see the errors. I wanna remove it from the other repos too.